dashboard/src/pages/login/CoreConnect.tsx

```diff
@@ -142,12 +151,14 @@ const CoreConnect = () => {
             autoComplete={DISABLE_AUTOFILL}
           >
             {status && (
-              <WarningAlert>
-                <p>
-                  <b>{status.error}</b>: Please ensure your fields are filled
-                  out correctly.
-                </p>
-              </WarningAlert>
+              <div className="mt-[-35px]">
```

Comment:
It would be better if this is in `rem`, or a builtin tailwind `-mt-9` if that works
also same as above, would be better if we just put the classname on the WarningAlert